.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  env: {
+    es6: true
+  },
+  parserOptions: {
+    ecmaVersion: 8,
+    sourceType: "module"
+  },
+  rules: {
+    indent: ["error", 2],
+    "linebreak-style": ["error", "unix"],
```

Comment:
If you are enforcing a linebreak style then a `.gitattributes` file is also needed for Window's users where git's setting for line-endings is going to default to `CRLF` on checkout.

Reply:
Would you prefer removing the linebreak style check in lint or adding a gitattributes file ? Seems like most of the code base has CRLF

Reply:
You need both. Git doesn't really have anything to do with linting but the opposite is not true. Linting will give errors if you checkout this repo on windows without a .gitattributes file and the eslint rc line-endings are set to LF.

But the files are actually LF. If you run `git ls-files --eol` then you will see that the index (the first column of output) shows all to be LF.

Reply:
My suggestion is to break as little as possible.

If we swap to unix linting then you must add a .gitattributes file for windows users so they checkout as LF. This would require users to reclone their repos.

That file would look like this:
```gitattributes
* text=auto eol=lf
```

Or keep rules without linebreak-style but make sure they are converted to LF when added to the index

```gitattributes
* text=auto
```

I think the least intrusive option is the 2nd as it doesn't require everyone to reclone. Unless there is a way to get the line-endings fixed without cloning.